### PR TITLE
P20-343: Modularize i18n sync in resources

### PR DESCRIPTION
**Pull request by artem-vavilov**
1. Modularized Apps i18n resource sync in
2. Modularized Dashboard i18n resource sync in
3. Modularized Pegasus i18n resource sync in
4. Moved files copying from `bin/i18n-codeorg/in.sh` to i18n resource modules
5. Deleted unused `bin/i18n-codeorg` scripts

## Links
- jira ticket: [P20-343](https://codedotorg.atlassian.net/browse/P20-343)

## Sync-in
Screenshot 2023-08-15 at 12 26 15

### CODE REVIEW

File: bin/i18n/resources/dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        CourseOfferings.sync_in
+        Courses.sync_in
+        CurriculumContent.sync_in
+        Docs.sync_in
+        Scripts.sync_in
+        SharedFunctions.sync_in
+        Standards.sync_in
+
+        puts 'Copying Dashboard source files'
+        FileUtils.mkdir_p(I18N_SOURCE_DIR_PATH)
```
Comment (mgc1194) [inline bin/i18n/resources/dashboard.rb]: Can we move the `FileUtils.mkdir_p(I18N_SOURCE_DIR_PATH)` to the beginning of the sync_in method?

**Review by mgc1194 — APPROVED** [review]
A bit long Output since I tested the whole sync-in, but it was successful
Screenshot 2023-08-15 at 12 26 15

```
➜  code-dot-org git:(P20-343/i18n-files-copying-sync-in-tests) ✗ ./bin/i18n/sync-in.rb                       git:(P20-343/i18n-files-copying-sync-in-tests|…1⚑21 
GetSecretValue: development/cdo/slack_bot_token
Sync in starting
There was a problem reading the existing manifest.  Rebuilding from scratch...
aPreparing external sources
Preparing *labs files
Redacting *labs content
Preparing blocks content
Redacting blocks content
Preparing level content
Preparing project content
Redacting level content
Preparing course offerings
Preparing dashboard courses
Redacting dashboard courses
Sync in curriculum content
[Destination directory for script is attempting to change] Script 20-hour wants to output strings to ../curriculum_content/unversioned/csf/20-hour.json, but other/20-hour.json already exists
[Destination directory for script is attempting to change] Script Hour of Code wants to output strings to ../curriculum_content/unversioned/hour-of-code/Hour of Code.json, but other/Hour of Code.json already exists
[Destination directory for script is attempting to change] Script K5-OnlinePD wants to output strings to ../curriculum_content/2018/k5-onlinepd/K5-OnlinePD.json, but 2018/K5-OnlinePD.json already exists
[Destination directory for script is attempting to change] Script allthethings wants to output strings to ../curriculum_content/unversioned/allthethingscourse/allthethings.json, but other/allthethings.json already exists
[Destination directory for script is attempting to change] Script applab-1hour wants to output strings to ../curriculum_content/unversioned/applab-1hour/applab-1hour.json, but other/applab-1hour.json already exists
[Destination directory for script is attempting to change] Script applab-2hour wants to output strings to ../curriculum_content/unversioned/applab-2hour/applab-2hour.json, but other/applab-2hour.json already exists
[Destination directory for script is attempting to change] Script applab-intro wants to output strings to ../curriculum_content/Hour of Code/applab-intro.json, but Hour of Code/applab-intro.json already exists
[Destination directory for script is attempting to change] Script aquatic wants to output strings to ../curriculum_content/Hour of Code/aquatic.json, but Hour of Code/aquatic.json already exists
[Destination directory for script is attempting to change] Script artist wants to output strings to ../curriculum_content/Hour of Code/artist.json, but Hour of Code/artist.json already exists
[Destination directory for script is attempting to change] Script basketball wants to output strings to ../curriculum_content/Hour of Code/basketball.json, but Hour of Code/basketball.json already exists
[Destination directory for script is attempting to change] Script course1 wants to output strings to ../curriculum_content/unversioned/csf/course1.json, but other/course1.json already exists
[Destination directory for script is attempting to change] Script course2 wants to output strings to ../curriculum_content/unversioned/csf/course2.json, but other/course2.json already exists
[Destination directory for script is attempting to change] Script course3 wants to output strings to ../curriculum_content/unversioned/csf/course3.json, but other/course3.json already exists
[Destination directory for script is attempting to change] Script course4 wants to output strings to ../curriculum_content/unversioned/csf/course4.json, but other/course4.json already exists
[Destination directory for script is attempting to change] Script coursea-2017 wants to output strings to ../curriculum_content/2017/csf/coursea-2017.json, but 2017/coursea-2017.json already exists
[Destination directory for script is attempting to change] Script coursea-2018 wants to output strings to ../curriculum_content/2018/csf/coursea-2018.json, but 2018/coursea-2018.json already exists
[Destination directory for script is attempting to change] Script coursea-2019 wants to output strings to ../curriculum_content/2019/csf/coursea-2019.json, but 2019/coursea-2019.json already exists
[Destination directory for script is attempting to change] Script coursea-2020 wants to output strings to ../curriculum_content/2020/csf/coursea-2020.json, but 2020/coursea-2020.json already exists
[Destination directory for script is attempting to change] Script coursea-2021 wants to output strings to ../curriculum_content/2021/csf/coursea-2021.json, but 2021/coursea-2021.json already exists
[Destination directory for script is attempting to change] Script coursea-2022 wants to output strings to ../curriculum_content/2022/csf/coursea-2022.json, but 2022/coursea-2022.json already exists
[Destination directory for script is attempting to change] Script courseb-2017 wants to output strings to ../curriculum_content/2017/csf/courseb-2017.json, but 2017/courseb-2017.json already exists
[Destination directory for script is attempting to change] Script courseb-2018 wants to output strings to ../curriculum_content/2018/csf/courseb-2018.json, but 2018/courseb-2018.json already exists
[Destination directory for script is attempting to change] Script courseb-2019 wants to output strings to ../curriculum_content/2019/csf/courseb-2019.json, but 2019/courseb-2019.json already exists
[Destination directory for script is attempting to change] Script courseb-2020 wants to output strings to ../curriculum_content/2020/csf/courseb-2020.json, but 2020/courseb-2020.json already exists
[Destination directory for script is attempting to change] Script courseb-2021 wants to output strings to ../curriculum_content/2021/csf/courseb-2021.json, but 2021/courseb-2021.json already exists
[Destination directory for script is attempting to change] Script courseb-2022 wants to output strings to ../curriculum_content/2022/csf/courseb-2022.json, but 2022/courseb-2022.json already exists
[Destination directory for script is attempting to change] Script coursec-2017 wants to output strings to ../curriculum_content/2017/csf/coursec-2017.json, but 2017/coursec-2017.json already exists
[Destination directory for script is attempting to change] Script coursec-2018 wants to output strings to ../curriculum_content/2018/csf/coursec-2018.json, but 2018/coursec-2018.json already exists
[Destination directory for script is attempting to change] Script coursec-2019 wants to output strings to ../curriculum_content/2019/csf/coursec-2019.json, but 2019/coursec-2019.json already exists
[Destination directory for script is attempting to change] Script coursec-2020 wants to output strings to ../curriculum_content/2020/csf/coursec-2020.json, but 2020/coursec-2020.json already exists
[Destination directory for script is attempting to change] Script coursec-2021 wants to output strings to ../curriculum_content/2021/csf/coursec-2021.json, but 2021/coursec-2021.json already exists
[Destination directory for script is attempting to change] Script coursec-2022 wants to output strings to ../curriculum_content/2022/csf/coursec-2022.json, but 2022/coursec-2022.json already exists
[Destination directory for script is attempting to change] Script coursed-2017 wants to output strings to ../curriculum_content/2017/csf/coursed-2017.json, but 2017/coursed-2017.json already exists
[Destination directory for script is attempting to change] Script coursed-2018 wants to output strings to ../curriculum_content/2018/csf/coursed-2018.json, but 2018/coursed-2018.json already exists
[Destination directory for script is attempting to change] Script coursed-2019 wants to output strings to ../curriculum_content/2019/csf/coursed-2019.json, but 2019/coursed-2019.json already exists
[Destination directory for script is attempting to change] Script coursed-2020 wants to output strings to ../curriculum_content/2020/csf/coursed-2020.json, but 2020/coursed-2020.json already exists
[Destination directory for script is attempting to change] Script coursed-2021 wants to output strings to ../curriculum_content/2021/csf/coursed-2021.json, but 2021/coursed-2021.json already exists
[Destination directory for script is attempting to change] Script coursed-2022 wants to output strings to ../curriculum_content/2022/csf/coursed-2022.json, but 2022/coursed-2022.json already exists
[Destination directory for script is attempting to change] Script coursee-2017 wants to output strings to ../curriculum_content/2017/csf/coursee-2017.json, but 2017/coursee-2017.json already exists
[Destination directory for script is attempting to change] Script coursee-2018 wants to output strings to ../curriculum_content/2018/csf/coursee-2018.json, but 2018/coursee-2018.json already exists
[Destination directory for script is attempting to change] Script coursee-2019 wants to output strings to ../curriculum_content/2019/csf/coursee-2019.json, but 2019/coursee-2019.json already exists
[Destination directory for script is attempting to change] Script coursee-2020 wants to output strings to ../curriculum_content/2020/csf/coursee-2020.json, but 2020/coursee-2020.json already exists
[Destination directory for script is attempting to change] Script coursee-2021 wants to output strings to ../curriculum_content/2021/csf/coursee-2021.json, but 2021/coursee-2021.json already exists
[Destination directory for script is attempting to change] Script coursee-2022 wants to output strings to ../curriculum_content/2022/csf/coursee-2022.json, but 2022/coursee-2022.json already exists
[Destination directory for script is attempting to change] Script coursef-2017 wants to output strings to ../curriculum_content/2017/csf/coursef-2017.json, but 2017/coursef-2017.json already exists
[Destination directory for script is attempting to change] Script coursef-2018 wants to output strings to ../curriculum_content/2018/csf/coursef-2018.json, but 2018/coursef-2018.json already exists
[Destination directory for script is attempting to change] Script coursef-2019 wants to output strings to ../curriculum_content/2019/csf/coursef-2019.json, but 2019/coursef-2019.json already exists
[Destination directory for script is attempting to change] Script coursef-2020 wants to output strings to ../curriculum_content/2020/csf/coursef-2020.json, but 2020/coursef-2020.json already exists
[Destination directory for script is attempting to change] Script coursef-2021 wants to output strings to ../curriculum_content/2021/csf/coursef-2021.json, but 2021/coursef-2021.json already exists
[Destination directory for script is attempting to change] Script coursef-2022 wants to output strings to ../curriculum_content/2022/csf/coursef-2022.json, but 2022/coursef-2022.json already exists
[Destination directory for script is attempting to change] Script csd-post-survey-2018 wants to output strings to ../curriculum_content/2018/csd/csd-post-survey-2018.json, but other/csd-post-survey-2018.json already exists
[Destination directory for script is attempting to change] Script csd-post-survey wants to output strings to ../curriculum_content/unversioned/csd-post-survey/csd-post-survey.json, but other/csd-post-survey.json already exists
[Destination directory for script is attempting to change] Script csd1-2017 wants to output strings to ../curriculum_content/2017/csd/csd1-2017.json, but 2017/csd1-2017.json already exists
[Destination directory for script is attempting to change] Script csd1-2018 wants to output strings to ../curriculum_content/2018/csd/csd1-2018.json, but 2018/csd1-2018.json already exists
[Destination directory for script is attempting to change] Script csd1-2019 wants to output strings to ../curriculum_content/2019/csd/csd1-2019.json, but 2019/csd1-2019.json already exists
[Destination directory for script is attempting to change] Script csd1-2021 wants to output strings to ../curriculum_content/2021/csd/csd1-2021.json, but other/csd1-2021.json already exists
[Destination directory for script is attempting to change] Script csd1-2022 wants to output strings to ../curriculum_content/2022/csd/csd1-2022.json, but 2022/csd1-2022.json already exists
[Destination directory for script is attempting to change] Script csd2-2017 wants to output strings to ../curriculum_content/2017/csd/csd2-2017.json, but 2017/csd2-2017.json already exists
[Destination directory for script is attempting to change] Script csd2-2018 wants to output strings to ../curriculum_content/2018/csd/csd2-2018.json, but 2018/csd2-2018.json already exists
[Destination directory for script is attempting to change] Script csd2-2019 wants to output strings to ../curriculum_content/2019/csd/csd2-2019.json, but 2019/csd2-2019.json already exists
[Destination directory for script is attempting to change] Script csd2-2021 wants to output strings to ../curriculum_content/2021/csd/csd2-2021.json, but other/csd2-2021.json already exists
[Destination directory for script is attempting to change] Script csd2-2022 wants to output strings to ../curriculum_content/2022/csd/csd2-2022.json, but 2022/csd2-2022.json already exists
[Destination directory for script is attempting to change] Script csd3-2017 wants to output strings to ../curriculum_content/2017/csd/csd3-2017.json, but 2017/csd3-2017.json already exists
[Destination directory for script is attempting to change] Script csd3-2018 wants to output strings to ../curriculum_content/2018/csd/csd3-2018.json, but 2018/csd3-2018.json already exists
[Destination directory for script is attempting to change] Script csd3-2019 wants to output strings to ../curriculum_content/2019/csd/csd3-2019.json, but 2019/csd3-2019.json already exists
[Destination directory for script is attempting to change] Script csd3-2021 wants to output strings to ../curriculum_content/2021/csd/csd3-2021.json, but other/csd3-2021.json already exists
[Destination directory for script is attempting to change] Script csd3-2022 wants to output strings to ../curriculum_content/2022/csd/csd3-2022.json, but 2022/csd3-2022.json already exists
[Destination directory for script is attempting to change] Script csd4-2017 wants to output strings to ../curriculum_content/2017/csd/csd4-2017.json, but 2017/csd4-2017.json already exists
[Destination directory for script is attempting to change] Script csd4-2018 wants to output strings to ../curriculum_content/2018/csd/csd4-2018.json, but 2018/csd4-2018.json already exists
[Destination directory for script is attempting to change] Script csd4-2019 wants to output strings to ../curriculum_content/2019/csd/csd4-2019.json, but 2019/csd4-2019.json already exists
[Destination directory for script is attempting to change] Script csd4-2021 wants to output strings to ../curriculum_content/2021/csd/csd4-2021.json, but other/csd4-2021.json already exists
[Destination directory for script is attempting to change] Script csd4-2022 wants to output strings to ../curriculum_content/2022/csd/csd4-2022.json, but 2022/csd4-2022.json already exists
[Destination directory for script is attempting to change] Script csd5-2017 wants to output strings to ../curriculum_content/2017/csd/csd5-2017.json, but 2017/csd5-2017.json already exists
[Destination directory for script is attempting to change] Script csd5-2018 wants to output strings to ../curriculum_content/2018/csd/csd5-2018.json, but 2018/csd5-2018.json already exists
[Destination directory for script is attempting to change] Script csd5-2019 wants to output strings to ../curriculum_content/2019/csd/csd5-2019.json, but 2019/csd5-2019.json already exists
[Destination directory for script is attempting to change] Script csd5-2021 wants to output strings to ../curriculum_content/2021/csd/csd5-2021.json, but other/csd5-2021.json already exists
[Destination directory for script is attempting to change] Script csd5-2022 wants to output strings to ../curriculum_content/2022/csd/csd5-2022.json, but 2022/csd5-2022.json already exists
[Destination directory for script is attempting to change] Script csd6-2017 wants to output strings to ../curriculum_content/2017/csd/csd6-2017.json, but 2017/csd6-2017.json already exists
[Destination directory for script is attempting to change] Script csd6-2018 wants to output strings to ../curriculum_content/2018/csd/csd6-2018.json, but 2018/csd6-2018.json already exists
[Destination directory for script is attempting to change] Script csd6-2019 wants to output strings to ../curriculum_content/2019/csd/csd6-2019.json, but 2019/csd6-2019.json already exists
[Destination directory for script is attempting to change] Script csd6-2021 wants to output strings to ../curriculum_content/2021/csd/csd6-2021.json, but 2021/csd6-2021.json already exists
[Destination directory for script is attempting to change] Script csd6-2022 wants to output strings to ../curriculum_content/2022/csd/csd6-2022.json, but other/csd6-2022.json already exists
[Destination directory for script is attempting to change] Script csd7-2021 wants to output strings to ../curriculum_content/2021/csd/csd7-2021.json, but other/csd7-2021.json already exists
[Destination directory for script is attempting to change] Script csd7-2022 wants to output strings to ../curriculum_content/2022/csd/csd7-2022.json, but 2022/csd7-2022.json already exists
[Destination directory for script is attempting to change] Script dance-2019 wants to output strings to ../curriculum_content/Hour of Code/dance-2019.json, but Hour of Code/dance-2019.json already exists
[Destination directory for script is attempting to change] Script dance-extras-2019 wants to output strings to ../curriculum_content/Hour of Code/dance-extras-2019.json, but Hour of Code/dance-extras-2019.json already exists
[Destination directory for script is attempting to change] Script dance-extras wants to output strings to ../curriculum_content/Hour of Code/dance-extras.json, but Hour of Code/dance-extras.json already exists
[Destination directory for script is attempting to change] Script dance wants to output strings to ../curriculum_content/Hour of Code/dance.json, but Hour of Code/dance.json already exists
[Destination directory for script is attempting to change] Script deepdive-debugging wants to output strings to ../curriculum_content/unversioned/deepdive-debugging/deepdive-debugging.json, but other/deepdive-debugging.json already exists
[Destination directory for script is attempting to change] Script express-2017 wants to output strings to ../curriculum_content/2017/csf/express-2017.json, but 2017/express-2017.json already exists
[Destination directory for script is attempting to change] Script express-2018 wants to output strings to ../curriculum_content/2018/csf/express-2018.json, but 2018/express-2018.json already exists
[Destination directory for script is attempting to change] Script express-2019 wants to output strings to ../curriculum_content/2019/csf/express-2019.json, but 2019/express-2019.json already exists
[Destination directory for script is attempting to change] Script express-2020 wants to output strings to ../curriculum_content/2020/csf/express-2020.json, but 2020/express-2020.json already exists
[Destination directory for script is attempting to change] Script express-2021 wants to output strings to ../curriculum_content/2021/csf/express-2021.json, but 2021/express-2021.json already exists
[Destination directory for script is attempting to change] Script express-2022 wants to output strings to ../curriculum_content/2022/csf/express-2022.json, but 2022/express-2022.json already exists
[Destination directory for script is attempting to change] Script flappy wants to output strings to ../curriculum_content/Hour of Code/flappy.json, but Hour of Code/flappy.json already exists
[Destination directory for script is attempting to change] Script frequency_analysis wants to output strings to ../curriculum_content/unversioned/frequency-analysis/frequency_analysis.json, but other/frequency_analysis.json already exists
[Destination directory for script is attempting to change] Script frozen wants to output strings to ../curriculum_content/Hour of Code/frozen.json, but Hour of Code/frozen.json already exists
[Destination directory for script is attempting to change] Script gamelab wants to output strings to ../curriculum_content/unversioned/gamelab/gamelab.json, but other/gamelab.json already exists
[Destination directory for script is attempting to change] Script gumball wants to output strings to ../curriculum_content/Hour of Code/gumball.json, but Hour of Code/gumball.json already exists
[Destination directory for script is attempting to change] Script hello-world-animals-2021 wants to output strings to ../curriculum_content/Hour of Code/hello-world-animals-2021.json, but Hour of Code/hello-world-animals-2021.json already exists
[Destination directory for script is attempting to change] Script hello-world-emoji-2021 wants to output strings to ../curriculum_content/Hour of Code/hello-world-emoji-2021.json, but Hour of Code/hello-world-emoji-2021.json already exists
[Destination directory for script is attempting to change] Script hello-world-food-2021 wants to output strings to ../curriculum_content/Hour of Code/hello-world-food-2021.json, but Hour of Code/hello-world-food-2021.json already exists
[Destination directory for script is attempting to change] Script hello-world-retro-2021 wants to output strings to ../curriculum_content/Hour of Code/hello-world-retro-2021.json, but Hour of Code/hello-world-retro-2021.json already exists
[Destination directory for script is attempting to change] Script hello-world-soccer-2022 wants to output strings to ../curriculum_content/Hour of Code/hello-world-soccer-2022.json, but Hour of Code/hello-world-soccer-2022.json already exists
[Destination directory for script is attempting to change] Script hello-world-space-2022 wants to output strings to ../curriculum_content/Hour of Code/hello-world-space-2022.json, but Hour of Code/hello-world-space-2022.json already exists
[Destination directory for script is attempting to change] Script hero wants to output strings to ../curriculum_content/Hour of Code/hero.json, but Hour of Code/hero.json already exists
[Destination directory for script is attempting to change] Script hoc-encryption wants to output strings to ../curriculum_content/Hour of Code/hoc-encryption.json, but Hour of Code/hoc-encryption.json already exists
[Destination directory for script is attempting to change] Script hourofcode wants to output strings to ../curriculum_content/Hour of Code/hourofcode.json, but Hour of Code/hourofcode.json already exists
[Destination directory for script is attempting to change] Script iceage wants to output strings to ../curriculum_content/Hour of Code/iceage.json, but Hour of Code/iceage.json already exists
[Destination directory for script is attempting to change] Script infinity wants to output strings to ../curriculum_content/Hour of Code/infinity.json, but Hour of Code/infinity.json already exists
[Destination directory for script is attempting to change] Script jigsaw wants to output strings to ../curriculum_content/unversioned/jigsaw/jigsaw.json, but other/jigsaw.json already exists
[Destination directory for script is attempting to change] Script k1hoc2017 wants to output strings to ../curriculum_content/unversioned/k1hoc2017/k1hoc2017.json, but other/k1hoc2017.json already exists
[Destination directory for script is attempting to change] Script k5-onlinepd-2019 wants to output strings to ../curriculum_content/2019/k5-onlinepd/k5-onlinepd-2019.json, but 2019/k5-onlinepd-2019.json already exists
[Destination directory for script is attempting to change] Script kodea-pd-2021 wants to output strings to ../curriculum_content/2021/kodea-pd-2021/kodea-pd-2021.json, but 2021/kodea-pd-2021.json already exists
[Destination directory for script is attempting to change] Script mc wants to output strings to ../curriculum_content/Hour of Code/mc.json, but Hour of Code/mc.json already exists
[Destination directory for script is attempting to change] Script minecraft wants to output strings to ../curriculum_content/Hour of Code/minecraft.json, but Hour of Code/minecraft.json already exists
[Destination directory for script is attempting to change] Script netsim wants to output strings to ../curriculum_content/unversioned/netsim/netsim.json, but other/netsim.json already exists
[Destination directory for script is attempting to change] Script oceans wants to output strings to ../curriculum_content/Hour of Code/oceans.json, but Hour of Code/oceans.json already exists
[Destination directory for script is attempting to change] Script odometer wants to output strings to ../curriculum_content/unversioned/odometer/odometer.json, but other/odometer.json already exists
[Destination directory for script is attempting to change] Script outbreak wants to output strings to ../curriculum_content/Hour of Code/outbreak.json, but Hour of Code/outbreak.json already exists
[Destination directory for script is attempting to change] Script pixelation wants to output strings to ../curriculum_content/unversioned/pixelation/pixelation.json, but other/pixelation.json already exists
[Destination directory for script is attempting to change] Script playlab wants to output strings to ../curriculum_content/Hour of Code/playlab.json, but Hour of Code/playlab.json already exists
[Destination directory for script is attempting to change] Script poem-art-2021 wants to output strings to ../curriculum_content/Hour of Code/poem-art-2021.json, but Hour of Code/poem-art-2021.json already exists
[Destination directory for script is attempting to change] Script poetry-hoc3 wants to output strings to ../curriculum_content/unversioned/poetry-hoc3/poetry-hoc3.json, but other/poetry-hoc3.json already exists
[Destination directory for script is attempting to change] Script pre-express-2017 wants to output strings to ../curriculum_content/2017/csf/pre-express-2017.json, but 2017/pre-express-2017.json already exists
[Destination directory for script is attempting to change] Script pre-express-2018 wants to output strings to ../curriculum_content/2018/csf/pre-express-2018.json, but 2018/pre-express-2018.json already exists
[Destination directory for script is attempting to change] Script pre-express-2019 wants to output strings to ../curriculum_content/2019/csf/pre-express-2019.json, but 2019/pre-express-2019.json already exists
[Destination directory for script is attempting to change] Script pre-express-2020 wants to output strings to ../curriculum_content/2020/csf/pre-express-2020.json, but 2020/pre-express-2020.json already exists
[Destination directory for script is attempting to change] Script pre-express-2021 wants to output strings to ../curriculum_content/2021/csf/pre-express-2021.json, but 2021/pre-express-2021.json already exists
[Destination directory for script is attempting to change] Script pre-express-2022 wants to output strings to ../curriculum_content/2022/csf/pre-express-2022.json, but 2022/pre-express-2022.json already exists
[Destination directory for script is attempting to change] Script sports wants to output strings to ../curriculum_content/Hour of Code/sports.json, but Hour of Code/sports.json already exists
[Destination directory for script is attempting to change] Script starwars wants to output strings to ../curriculum_content/Hour of Code/starwars.json, but Hour of Code/starwars.json already exists
[Destination directory for script is attempting to change] Script starwarsblocks wants to output strings to ../curriculum_content/Hour of Code/starwarsblocks.json, but Hour of Code/starwarsblocks.json already exists
[Destination directory for script is attempting to change] Script text-compression wants to output strings to ../curriculum_content/Hour of Code/text-compression.json, but Hour of Code/text-compression.json already exists
[Destination directory for script is attempting to change] Script vigenere wants to output strings to ../curriculum_content/unversioned/vigenere/vigenere.json, but other/vigenere.json already exists
Preparing docs content
Preparation of /Users/mariogilcorrea/RubymineProjects/code-dot-org/i18n/locales/source/docs/gamelab.json
Redaction of /Users/mariogilcorrea/RubymineProjects/code-dot-org/i18n/locales/source/docs/gamelab.json
Preparation of /Users/mariogilcorrea/RubymineProjects/code-dot-org/i18n/locales/source/docs/applab.json
Redaction of /Users/mariogilcorrea/RubymineProjects/code-dot-org/i18n/locales/source/docs/applab.json
Preparation of /Users/mariogilcorrea/RubymineProjects/code-dot-org/i18n/locales/source/docs/weblab.json
Redaction of /Users/mariogilcorrea/RubymineProjects/code-dot-org/i18n/locales/source/docs/weblab.json
Preparing dashboard scripts
Redacting dashboard scripts
Preparing shared functions
Preparing standards content
Copying Dashboard source files
Preparing Hour of Code content
public/csforgood.md does not exist
public/educate/csc.md.partial does not exist
public/hourofcode/artist.md.partial does not exist
public/hourofcode/flappy.md.partial does not exist
public/hourofcode/frozen.md.partial does not exist
public/hourofcode/hourofcode.md.partial does not exist
public/hourofcode/infinity.md.partial does not exist
public/hourofcode/mc.md.partial does not exist
public/hourofcode/playlab.md.partial does not exist
public/hourofcode/starwars.md.partial does not exist
public/hourofcode/unplugged-conditionals-with-cards.md.partial does not exist
Copying Pegasus source file
Sync in completed successfully

```

**Review by cat5inthecradle — APPROVED** [review]
Infra was tagged because we're codeowners of the .github directory. The changes there look fine.

File: bin/i18n/sync-in.rb [hunk 2]
```diff
@@ -11,22 +11,9 @@ module I18n
...
-      I18n::Resources::Apps::ExternalSources.sync_in
-      I18n::Resources::Dashboard::Scripts.sync_in
-      I18n::Resources::Dashboard::Courses.sync_in
-      I18n::Resources::Apps::Labs.sync_in
-      I18n::Resources::Pegasus::Markdown.sync_in
-      puts "Copying source files"
-      I18nScriptUtils.run_bash_script "bin/i18n-codeorg/in.sh"
+      I18n::Resources::Apps.sync_in
+      I18n::Resources::Dashboard.sync_in
+      I18n::Resources::Pegasus.sync_in
```
Comment (Hamms) [inline bin/i18n/sync-in.rb:16]: This is so clean and intuitive now!

File: bin/test/animation_assets/test_manifest_builder.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@ def setup
   def test_get_animation_strings_for_spritelab_quietly
     animation_strings = nil
 
-    VCR.use_cassette('animations/manifest_spritelab_strings') do
+    VCR.use_cassette('animations/manifest_spritelab_strings', record: :none) do
```
Comment (Hamms) [inline bin/test/animation_assets/test_manifest_builder.rb:13]: why is this change necessary?

**Review by Hamms — APPROVED** [review]
This is fantastic to see! So much cleaner and more consistent, and no more perl!

It's been some time since I've regularly worked with this code, so if there are any surprises or weird edge cases here I have low confidence in my ability to catch them. But, given that, everything here looks good to me; thank you for such a comprehensive cleanup!

File: bin/test/animation_assets/test_manifest_builder.rb (hunk 3 above)
Comment (artem-vavilov) [reply bin/test/animation_assets/test_manifest_builder.rb:13]: The vcs cassette for this test was created manually to cover all the code conditions at once, so it should never be rewritten by the real response
